blocks/header/header.css

```diff
@@ -0,0 +1,46 @@
+body
+{
+    width: 1673px;
+    margin: 0 auto;
+
+    background-color: white;
+}
+.header
+{
+    z-index: 1;
+
+    width: 100%;
+    height: 648px;
+
+    background-image: url(../../assets/img/header/sochi-2258855.png);
+}
+.header__logo
+{
+    margin-top: 21px;
+    margin-left: 789px;

```

Comment:
Once you have a smaller screen, this will no longer be in the middle of a page.

Let's consider another approach.